tests/unit/api/serialization/test_label_mapper.py

```diff
@@ -374,3 +374,30 @@ def test_label_graph_backward(self):
         }
         with pytest.raises(ValueError):
             LabelGraphMapper.backward(instance=forward, all_labels=labels)
+        # Checking label deletion case
```

Comment:
This test looks nice, however, currently we didn't check the `unit/api` test. we need to re-introduce.